platforms/jvm/toolchains-jvm/src/main/java/org/gradle/jvm/toolchain/JvmToolchainManagement.java

```diff
@@ -38,7 +38,7 @@ public interface JvmToolchainManagement {
      * {@link org.gradle.api.NamedDomainObjectList} based handler for configuring an
      * ordered collection of Java toolchain repositories:
      *
-     * <pre class='autoTested'>
+     * <pre>
```

Comment:
This test wasn't run, and is broken, don't auto-test it.

Reply:
So it is only broken in the sense that it doesn't compile because it sets custom resolver classes that aren't defined.  It is still a valid partial example, that shows proper syntax and demonstrates the concept, like many other blocks of javadoc.  

I think it's fine to leave it, just not to compile it.